packages/form-js-viewer/src/render/components/form-fields/Textfield.js

```diff
@@ -68,4 +68,4 @@ Textfield.type = type;
 Textfield.label = 'Text Field';
 Textfield.keyed = true;
 Textfield.emptyValue = '';
-Textfield.sanitizeValue = ({ value }) => (isArray(value) || isObject(value)) ? null : String(value);
\ No newline at end of file
+Textfield.sanitizeValue = ({ value }) => (isArray(value) || isObject(value)) ? '' : String(value);
```

Comment:
Can you describe why this change is needed? Do we have a test case for this?

Reply:
As I built out the `textArea`, I noticed that I was sanitizing the `textField` to a null value. This is however incorrect as we use empty string as our empty state for `textField`, never null. 

This addresses it and yes we do have a test case for it: https://github.com/bpmn-io/form-js/blob/183e8cd911da92c9bafe38e0206e06e27b8d03cc/packages/form-js-viewer/test/spec/import/Importer.spec.js#L326